server/src/main/java/org/apache/calcite/sql/ddl/SqlCreateView.java

```diff
@@ -86,6 +96,9 @@
     writer.keyword("AS");
     writer.newlineAndIndent();
     query.unparse(writer, 0, 0);
+    if (withCheckOption != null && withCheckOption.getValueAs(Boolean.class)) {
```

Comment:
nit: We don't need to check for null since we control what value withCheckOption is set to from within this same class, and it's never being set to null.